actors/power/tests/power_actor_tests.rs

```diff
@@ -1310,3 +1441,19 @@ mod submit_porep_for_bulk_verify_tests {
         expect_abort(ExitCode::USR_FORBIDDEN, h.submit_porep_for_bulk_verify(&mut rt, MINER, info));
     }
 }
+
+fn expect_query_network_info(rt: &mut MockRuntime, h: &Harness) {
```

Comment:
This function already exists on the harness, so please use that one nd remove this duplicate